### PR TITLE
Add initial list of robot packages

### PR DESCRIPTION
**Pull request by destogl**
Here just a first list of known packages to me...

@dignakov, @urrsk, @fmauch, @nbbrooks, @mcbed, @marcbone, @falfab
Is it OK to list your package here? Maybe we could add some pictures for the nicer site, what do you think? Any other ideas?

### CODE REVIEW

**Comment by dignakov** [comment]
Of course, if you would like my version of the KUKA drivers it's here:
https://github.com/dignakov/ros2_control_kuka_driver
(I'll make some screenshots this week)

PickNick has been doing awesome things with the ABB driver:
https://github.com/PickNikRobotics/abb_ros2

**Comment by fmauch** [comment]
Fine from my side, obviously!

**Comment by mcbed** [comment]
Of course it's fine for me !

File: doc/supported_robots/supported_robots.rst [hunk 1]
```diff
@@ -5,4 +5,10 @@ This page hosts a list of supported robots and references to them.
 To add your robot, submit a PR to this page on Github!
 
 .. * Use an unordered list
+Official (supported by robot manufacturer):
+* `Universal Robots <https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver>`_
+* `Franka Emika research robots <https://github.com/frankaemika/franka_ros2>`_
 
+Unofficial (from the community):
+* `KUKA industrial robots (KUKA Robot Sensor Interface (RSI)) <https://github.com/ICube-Robotics/iiwa_ros2>`_
+* `KUKA IIWA (KUKA Fast Robot Interface (FRI)) <https://github.com/ICube-Robotics/iiwa_ros2>`_
```
Comment (destogl) [inline doc/supported_robots/supported_robots.rst:14]:
```suggestion
* `KUKA IIWA (KUKA Fast Robot Interface (FRI)) <https://github.com/ICube-Robotics/iiwa_ros2>`_
* `ABB - EGM interface <https://github.com/PickNikRobotics/abb_ros2>`_
```

File: doc/supported_robots/supported_robots.rst [hunk 2]
```diff
@@ -5,4 +5,10 @@ This page hosts a list of supported robots and references to them.
 To add your robot, submit a PR to this page on Github!
 
 .. * Use an unordered list
+Official (supported by robot manufacturer):
+* `Universal Robots <https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver>`_
+* `Franka Emika research robots <https://github.com/frankaemika/franka_ros2>`_
 
+Unofficial (from the community):
+* `KUKA industrial robots (KUKA Robot Sensor Interface (RSI)) <https://github.com/ICube-Robotics/iiwa_ros2>`_
```
Comment (destogl) [inline doc/supported_robots/supported_robots.rst:13]:
```suggestion
* `KUKA industrial robots (KUKA Robot Sensor Interface (RSI)) <https://github.com/dignakov/ros2_control_kuka_driver>`_
```

**Comment by destogl** [comment]
@mcbed and @mhubii what's the difference between your `KUKA FRI` drivers? Are they alternative or something else. AFAIK - there should be difference between a driver for IIWA and LBR, although both have the sam name.

**Comment by nbbrooks** [comment]
:+1: although it seems like github [isn't rendering the lists correctly](https://github.com/ros-controls/control.ros.org/blob/add-some-initial-list-of-robots/doc/supported_robots/supported_robots.rst)

**Comment by mhubii** [comment]
hi @destogl, currently not in europe so response time might be slow. They appear very similar in nature and both use KUKA's FRI UDP communication library. Ill have a more detailed look later.

File: doc/supported_robots/supported_robots.rst [hunk 3]
```diff
@@ -5,4 +5,17 @@ This page hosts a list of supported robots and references to them.
...
+Unofficial (from the community):
+* `KUKA industrial robots (KUKA Robot Sensor Interface (RSI)) <https://github.com/dignakov/ros2_control_kuka_driver>`_
+* `KUKA IIWA (KUKA Fast Robot Interface (FRI)) <https://github.com/ICube-Robotics/iiwa_ros2>`_
+* `KUKA LBR and IIWA (KUKA Fast Robot Interface (FRI) <https://github.com/KCL-BMEIS/lbr_fri_ros2_stack>`_
+* `ABB - EGM interface <https://github.com/PickNikRobotics/abb_ros2>`_
+* `Mitsubishi RV1A <https://github.com/ICube-Robotics/mrv1a_ros2`_
+
+Non robot-devices:
+* `Force Dimension haptic devices <https://github.com/ICube-Robotics/forcedimension_ros2>`_
+* `
```
Comment (destogl) [inline doc/supported_robots/supported_robots.rst:21]:
```suggestion

Mobile manipulators:
* `Hello Robot - Stretch (for simple simulation with MoveIt2) <https://github.com/hello-robot/stretch_ros2/blob/galactic/stretch_moveit_config/README.md>`_
```

**Review by bmagyar — APPROVED** [review]

**Comment by nbbrooks** [comment]
I thought controls.ros.org might interpret the RST differently than here, but it looks like it also has an issue with the bulleting format - made a PR to fix [here](https://github.com/ros-controls/control.ros.org/pull/46)

**Pull request merged**